packages/odyssey-svgr/src/iconTemplate.ts

```diff
@@ -27,10 +27,8 @@ export const iconTemplate: Template = ({ componentName, jsx }, { tpl }) => {
   const fragmentJsx = jsxFragment(
     jsxOpeningFragment(),
     jsxClosingFragment(),
-    // @ts-expect-error TEMP: This type aren't working after the upgrade, but they need to be fixed.
```

Comment:
For this and other line removals like it, a patch bump in a library has cleared up these issues